### ISSUE TITLE
Question about Deepall DIAL implementation

### ISSUE THREAD

**Issue by Car-la-F** [marrlab/DomainLab#204]
I started working on the documentation for DIAL on the branch `cf_docu_DIAL`. Doing so I got some questions:

1. the paper proposes two loss functions, one using only the classical cross entropy loss and one using the Kullback-Leibler divergence. Can one control which loss is used in Domainlab?
2. The paper uses two hyperparameter ($r$ and $\lambda$) to weight the terms in the loss function. $r$ weights the domain classification loss and $\lambda$ the label classification loss in the adversary domains and the Kullback-Leibler divergence between the label output of natural domain and their adversary domains. In the code I can only find `gamma_reg` and I don't understand which term is seen as the regularisation term

https://github.com/marrlab/DomainLab/blob/0e64a4a2cf8bf30ec8d92de7b43964adf55f4f42/domainlab/algos/trainers/train_dial.py#L57

**Comment by Car-la-F** [comment]
loss.sum(): loss for original image
loss_dial.sum(): loss for augmented image
